source/common/http/http1/codec_impl.h

```diff
@@ -300,16 +300,16 @@ class ConnectionImpl : public virtual Connection, protected Logger::Loggable<Log
   HeaderParsingState header_parsing_state_{HeaderParsingState::Field};
   HeaderString current_header_field_;
   HeaderString current_header_value_;
-  bool reset_stream_called_{};
+  bool reset_stream_called_ : 1;
```

Comment:
You need to pack all of the bitfield bools next to each other to get the savings. If you need to just move them all to protected (it's not a big deal).